src/runner.js

```diff
@@ -35,7 +47,16 @@ export default async function runPerformanceTest (username, accessKey, argv, nam
         params: { rate: cpuRate }
     })
     await browser.url(site)
-    const result = await browser.assertPerformance(name, metrics)
-    await browser.deleteSession()
-    return { sessionId, result }
+
+    try {
+        const result = await browser.assertPerformance(name, metrics)
+        await browser.deleteSession()
+        return { sessionId, result }
+    } catch (e) {
+        await browser.deleteSession()
+
+        // log data couldn't be fetched due to a tracing issue
+        // run test again:
+        return runPerformanceTest(username, accessKey, argv, name, build, logDir)
```

Comment:
Should we somehow put a limit on how many times it can retry? Or it's on the user who should hit `CTRL-C`? If the latter, maybe we should print a log saying something like "retrying because of ..."`?

Reply:
I added a limit of max 3 retries